src/connectedGraph.js

```diff
@@ -0,0 +1,21 @@
+import setCompliment from '../src/setCompliment'
+import dataTypeCheck from '../src/dataTypeCheck'
+
+export default function connectedGraph(graph){
+
+  if(!dataTypeCheck(graph,'object')){
+    return "Wrong Data Type"
+  }
+  let allPoints = Object.keys(graph)
+  let connected = []
+  let toCheck = [allPoints[0]]
+
+  while(toCheck.length>0) {
```

Comment:
just a stylistic note...keeping spacing consistent throughout your program is good practice. and in general space between operators is easier to read and can reduce bugs :)